### DISCUSSION TITLE
Settings window can get horizontally smaller than its content (3.8.0rc)

### DISCUSSION THREAD

**Discussion by passing** [jamulussoftware/jamulus#1792] [Ideas]
**Describe the bug**
Settings window can get horizontally smaller than its content

**To Reproduce**
Open Settings, resize Window horizontally

**Expected behavior**
Window should not allow to get smaller than its content

**Screenshots**
Settings window in **jamulus_3.8.0beta2** minimal size:
![image](https://user-images.githubusercontent.com/8142186/120087237-81f10700-c0e6-11eb-8ad6-a7722ccd1d73.png)

Settings window in **jamulus_3.8.0rc1** & **jamulus_3.8.0rc2** minimal size:
![image](https://user-images.githubusercontent.com/8142186/120087259-ae0c8800-c0e6-11eb-9df2-30fdda73e77c.png)

**Operating system**
Linux (Xubuntu 20.04.2 LTS)

**Version of Jamulus**
jamulus_3.8.0rc1 & jamulus_3.8.0rc2

**Comment by dcorson-ticino-com** [comment]
This was requested and added recently.

**Comment by passing** [comment]
oh ok, I didn't expect this is on purpose!

**Comment by hoffie** [comment]
Thanks for the report.
First of all, as this is something which the user can control, I'd say that this is nothing which should stop 3.8.0, right?

I don't have a strong opinion on the described behavior. The extreme example from above obviously doesn't make sense. One might argue that all window sizes which cut off UI elements without scroll bars would be hard to justify?

**Comment by gilgongo** [comment]
As this is I understand by design, moving to discussion so that any action can be agreed there.

**Comment by dcorson-ticino-com** [comment]
As I understood the request it was to have a slim window to be able to see the ping and delay.
Maybe with the ping and delay now on the clientdlg we could revert this in 3.8.1.
